node/flatpak-node-generator.py

```diff
@@ -582,7 +582,7 @@ def add_archive_source(self,
         source: Dict[str, Any] = {
             'type': 'archive',
             'url': url,
-            'strip-components': 1,
+            'strip-components': strip_components,
```

Comment:
This is unrelated to the PR, just a tiny bugfix (I guess it currently doesn't affect anything).